src/Symfony/Component/Notifier/Bridge/KazInfoTeh/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "symfony/kazinfoteh-notifier",
+  "type": "symfony-bridge",
+  "description": "Symfony KazInfoTeh Notifier Bridge",
+  "keywords": ["KazInfoTeh", "notifier"],
+  "homepage": "https://symfony.com",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Egor Taranov",
+      "email": "dev@taranovegor.com",
+      "homepage": "https://taranovegor.com/contribution"
+    },
+    {
+      "name": "Symfony Community",
+      "homepage": "https://symfony.com/contributors"
+    }
+  ],
+  "require": {
+    "php": ">=7.2.5",
+    "symfony/http-client": "^4.3|^5.0",
+    "symfony/notifier": "^5.3"
```

Comment:
If this PR must target 6.1 branch, we should match the PHP version and in any case support Sf 6 for symfony/http-client and symfony/notifier

Reply:
fixed on
```
    "php": ">=8.0.2",
    "symfony/http-client": "^5.4|^6.0",
    "symfony/notifier": "^5.4|^6.0"
```

Reply:
versions of symfony replaced to `^6.1`